Orange/widgets/utils/itemmodels.py

```diff
@@ -222,39 +239,34 @@ def __setitem__(self, i, value):
     def wrap(self, table):
         self.beginResetModel()
         self._table = table
+        rows, cols = range(len(table)), range(len(table[0]) if table else 0)
+        self._roleData = [[{} for _ in cols] for _ in rows]
```

Comment:
I'm not sure this is a good idea. An empty dictionary takes 288 bytes on my system. A very modest table with 10 columns and 100 rows thus takes 288 KB for empty dicts. 10000 columns and 100 rows -- which is still not outrageously big -- will take 288.000,000 bytes. If `table` is, say, a numpy array of doubles, the data will take 8 MB and the `_roleData`, which is create automatically, just for the case we might need it, will take 288 MB.